### PR TITLE
Proof P12 #611

### PR DESCRIPTION
**Pull request by EGA-SUPREMO**
Checked for typos with the reduced quality scans.
I didn't check for the line breaks though..

### CODE REVIEW

File: Luminary099/P12.agc [hunk 1]
```diff
@@ -200,7 +200,7 @@ P12INIT		DLOAD			# INITIALIZE ENGINE DATA.  USED FOR P12 AND
 COMMINIT	DLOAD	DAD		# INITIALIZE TARGET DATA.  USED BY P12, P70
 			HINJECT		# AND P71 IF IT DOES NOT FOLLOW P70.
 			/LAND/
-		STODL	RCO
+		STODL	RCD
```
Comment (wopian) [inline Luminary099/P12.agc:203]: Definitely looks like `RCD`, will need to check the following Luminary099 files that define/reference the variable.

- ASCENT_GUIDANCE.agc L221, L222 and L576
- ERASABLE_ASSIGNMENT.agc L2614 (the variable definition) and L2615

Comment (wopian) [reply Luminary099/P12.agc:203]: From the high-res scans this is supposed to be `RCO` and not `RCD`, the O and D characters are incredible difficult to distinguish when not printed correctly.

ERASABLE_ASSIGNMENT variable definition: 
xnviewmp_37D2qtDkTT

Luminary210 with the same variable definition
image

File: Luminary099/P12.agc [hunk 2]
```diff
@@ -91,14 +91,14 @@ P12LMB		DLOAD
 		MXV	VSL6
 			REFSMMAT
 		STCALL	R		# COMPUTE R = POS(TIG)*2(-24) M.
-			MUNGRAV		# COMPUTE GDT1/2(TIG)*2(-T)M/CS.
+			MUNGRAV		# COMPUTE GDT1/2(TIG)*2(-7) M/CS.
```
[the remark this answers is not in the document]
Comment (EGA-SUPREMO) [reply Luminary099/P12.agc]: Ok, then I'll also remove the space from the 89 and 93 lines of code

File: Luminary099/P12.agc (hunk 1 above)
Comment (EGA-SUPREMO) [reply Luminary099/P12.agc]: Done!

**Pull request merged**